.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,10 @@
+<!-- Please use this template only for submitting feature requests or enhancements -->
```

Comment:
Add language to mention KEPs for larger, in-depth enhancements.

Also, in k/k, often issues get filed as parts of milestone features, so it might be a good idea to remind folks to link to the related feature in k/features? I'm a bit fuzzy on the process.

Reply:
> Also, in k/k, often issues get filed as parts of milestone features, so it might be a good idea to remind folks to link to the related feature in k/features?

k/features is more like an umbrella issue for features that have been approved by a SIG. Feature requests in k/k are more the enhancement-sort and are also requested by users. Most of the `kind/feature` issues do not really talk about features big enough or matured enough to have a related feature in k/features.

Also, from what I understand with @justaugustus's KEP, the idea is to rename k/features as k/keps. In that case, maybe we shouldn't point users to k/features...it might lead to a spam of feature requests there. :/

Reply:
Yup, that makes sense in my head. Which is why using enhancements instead of features also makes sense.

Reply:
xref https://github.com/kubernetes/sig-release/pull/298#issuecomment-422481846

Reply:
Instead: `Please only use this template for submitting enhancement requests`

Reply:
To add to the conversation:
- no need to mention KEPs here. Anyone submitting an enhancement request should not require intimate knowledge of PM process. We'll assume / hope that these submissions are vetted by someone more project adjacent to ensure they get routed correctly.
- k/features will be renamed to k/enhancements, to represent the union of work captured in both KEPs (multi-cycle work) and Features (approximately single-cycle work). Formal definitions forthcoming, but that's the general gist.

For anyone interested in reading more:
- KEP-1a: https://github.com/kubernetes/community/blob/master/keps/0001a-meta-kep-implementation.md
- KEP Implementation Umbrella: https://github.com/kubernetes/features/issues/617
- KEP Implementation Tracking board: https://github.com/orgs/kubernetes/projects/5